src/bufr/BufrParser/Query/Query.cpp

```diff
@@ -97,16 +96,15 @@ namespace bufr {
         maskCache_.insert({dataProvider_.getSubset(), masks});
     }
 
-    __details::Target Query::findTarget(const std::string &targetName,
-                                        const std::string &query) const {
+    std::shared_ptr<Target> Query::findTarget(const std::string &targetName,
```

Comment:
Looks like accesses to structures now go through a pointer. Why the change to a pointer?